mkdocs.yml

```
@@ -13,6 +13,7 @@ pages:
       - "Example Skills":
               - "Introduction": "examples/introduction.md"
               - "Weather": "examples/weather.md"
+              - "random": "examples/random.md"
```

This is the documentation index. Please use correct capitalisation and try to make it clear what the skill does in a short number of words. Perhaps like this.

```suggestion
              - "Random Chance": "examples/random.md"
```